{"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"? { ","kind":"text"},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}]}],"kind":"declarations"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/mapsnativesdk\/maperror\/errordescription"]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/MapsNativeSDK\/documentation\/MapsNativeSDK","doc:\/\/MapsNativeSDK\/documentation\/MapsNativeSDK\/MapError"]]},"abstract":[{"type":"text","text":"Inherited from "},{"code":"LocalizedError.errorDescription","type":"codeVoice"},{"type":"text","text":"."}],"metadata":{"roleHeading":"Instance Property","externalID":"s:13MapsNativeSDK8MapErrorO16errorDescriptionSSSgvp","modules":[{"name":"MapsNativeSDK"}],"title":"errorDescription","symbolKind":"property","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"role":"symbol"},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/MapsNativeSDK\/documentation\/MapsNativeSDK\/MapError\/errorDescription"},"references":{"doc://MapsNativeSDK/documentation/MapsNativeSDK":{"identifier":"doc:\/\/MapsNativeSDK\/documentation\/MapsNativeSDK","abstract":[{"type":"strong","inlineContent":[{"type":"text","text":"MapsNativeSDK"}]},{"text":" - библиотека, позволяющая использовать картографические данные и геосервисы VK в ваших iOS-приложениях.","type":"text"}],"role":"collection","type":"topic","kind":"symbol","title":"MapsNativeSDK","url":"\/documentation\/mapsnativesdk"},"doc://MapsNativeSDK/documentation/MapsNativeSDK/MapError/errorDescription":{"title":"errorDescription","abstract":[],"type":"topic","role":"symbol","identifier":"doc:\/\/MapsNativeSDK\/documentation\/MapsNativeSDK\/MapError\/errorDescription","kind":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"url":"\/documentation\/mapsnativesdk\/maperror\/errordescription"},"doc://MapsNativeSDK/documentation/MapsNativeSDK/MapError":{"kind":"symbol","identifier":"doc:\/\/MapsNativeSDK\/documentation\/MapsNativeSDK\/MapError","url":"\/documentation\/mapsnativesdk\/maperror","navigatorTitle":[{"text":"MapError","kind":"identifier"}],"title":"MapError","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"MapError"}],"type":"topic","abstract":[]}}}